driver/config/config_test.go

```diff
@@ -867,34 +888,38 @@ func TestLoadingTLSConfig(t *testing.T) {
 		p := config.MustNew(t, logger, os.Stderr, configx.SkipValidation())
 		p.MustSet(ctx, config.ViperKeyAdminTLSKeyPath, keyPath)
 		p.MustSet(ctx, config.ViperKeyAdminTLSCertPath, certPath)
-		assert.NotNil(t, p.GetTSLCertificatesForAdmin(ctx))
-		assert.Equal(t, "Setting up HTTPS for admin", hook.LastEntry().Message)
+		assert.NotNil(t, p.GetTLSCertificatesForAdmin(ctx))
+		assert.Equal(t, "Setting up HTTPS for admin (automatic certificate reloading active)", hook.LastEntry().Message)
 	})
 
 	t.Run("case=admin: failing to load inline base64 certificate", func(t *testing.T) {
 		logger := logrusx.New("", "")
-		logger.Logger.ExitFunc = func(code int) {}
+		exited := false
+		logger.Logger.ExitFunc = func(code int) { exited = true }
 		hook := new(test.Hook)
 		logger.Logger.Hooks.Add(hook)
 
 		p := config.MustNew(t, logger, os.Stderr, configx.SkipValidation())
 		p.MustSet(ctx, config.ViperKeyAdminTLSKeyBase64, "empty")
 		p.MustSet(ctx, config.ViperKeyAdminTLSCertBase64, certBase64)
-		assert.Nil(t, p.GetTSLCertificatesForAdmin(ctx))
-		assert.Equal(t, "TLS has not been configured for admin, skipping", hook.LastEntry().Message)
+		assert.Nil(t, p.GetTLSCertificatesForAdmin(ctx))
+		assert.Equal(t, "Unable to load HTTPS TLS Certificate", hook.LastEntry().Message)
+		assert.True(t, exited)
 	})
 
 	t.Run("case=admin: failing to load certificate from a file", func(t *testing.T) {
 		logger := logrusx.New("", "")
-		logger.Logger.ExitFunc = func(code int) {}
+		exited := false
+		logger.Logger.ExitFunc = func(code int) { exited = true }
 		hook := new(test.Hook)
 		logger.Logger.Hooks.Add(hook)
```

Comment:
You could make this a small local test helper function here, as it is used so often. Something like
```go 
func newTestLogger() (l *logrusx.Logger, h *test.Hook, exited *bool) {
  ...
}
```

Reply:
Added helper func